caffeine/src/main/java/com/percolate/caffeine/DialogUtils.java

```diff
@@ -31,13 +35,17 @@ public static AlertDialog quickDialog(final Activity context, final String messa
         final SpannableString s = new SpannableString(message); //Make links clickable
         Linkify.addLinks(s, Linkify.ALL);
 
-        Builder builder = new AlertDialog.Builder(context);
-        builder.setMessage(s);
-        builder.setPositiveButton(android.R.string.ok, closeDialogListener());
-        AlertDialog dialog = builder.create();
-        dialog.show();
-
-        ((TextView) dialog.findViewById(android.R.id.message)).setMovementMethod(LinkMovementMethod.getInstance()); //Make links clickable
+        if(dialog != null && dialog.isShowing()){
```

Comment:
I think there is another bug in these changes.  Since you made AlertDialog static, if you call `quickDialog` from Activity A, then open Activity B and call `quickDialog` nothing will be shown unless the user has closed the Dialog from Activity A.  I think we would need to keep a separate instance of AlertDialog for each `Activity context` that we get.